build.sbt

```diff
@@ -7,9 +7,10 @@ description := "ZIO wrapper for Scala DB libraries (e.g. Doobie)"
 
 val scala212Version = "2.12.15"
 val scala213Version = "2.13.8"
-val supportedScalaVersions = List(scala212Version, scala213Version)
+val scala3Version = "3.1.3"
+val supportedScalaVersions = List(scala212Version, scala213Version, scala3Version)
 
-scalaVersion := scala213Version
+scalaVersion := scala3Version
```

Comment:
Updated default version to scala3 but this might not be desirable given that editor supports are still trying to catch up. Happy to set it back to 2.13.